app/assets/stylesheets/module-admin.scss

```diff
@@ -418,6 +418,10 @@ body.gobierto_admin {
   margin-top: 220px;
 }
 
+.admin_side_actions .simulate-a {
```

Comment:
Why this? This does not make any sense. Please remove. And the corresponding call in the HTML.

Reply:
hi, this style is required to get and output uniform like below (note the gray color)

1. with this style
![Screenshot from 2021-07-23 09-40-58](https://user-images.githubusercontent.com/288355/126751653-ddec50ce-cbcb-4d66-abcd-c18fa34c6a32.png)

2. without force style
![Screenshot from 2021-07-23 09-40-41](https://user-images.githubusercontent.com/288355/126751667-b196c239-b018-441a-8b4a-d730205fdc20.png)

should remove or refactor class name?

thx